app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -56,6 +57,8 @@ class SelectionInteractionViewModel private constructor(
     computeChoiceItems(choiceSubtitledHtmls, hasConversationView, this)
 
   private val isAnswerAvailable = ObservableField(false)
+  val selectedItemText = ObservableField("Please select all correct choices")
+  val selectedItemsList: List<ObservableBoolean>? = null
```

Comment:
So what we want to model here isn't selected items, but rather selectableItems. This view model should set all checkboxes as enabled by default (true values), and then disable all unselected items once the max number is selected (setting just those ones to false).